frontend/src/old-pages/Users/Users.tsx

```diff
@@ -98,12 +81,15 @@ export default function Users(props: any) {
   }
 
   const createUser = () => {
-    userValidate()
-    const validated = getState([...errorsPath, 'validated'])
+    const validated = userValidate()
+    setCreateUserInputValidated(validated)
```

Comment:
suggestion(non-blocking): to do an extra mile what you can do is clear this flag as soon as the user types a new value, otherwise the input will stay red until a valid email is inserted and the button pressed

Reply:
Right, will remove the error in the onChange callback of the input field